packages/portal-proto/src/features/cDave/Controls.tsx

```diff
@@ -141,7 +141,11 @@ const FieldControl: React.FC<FieldControlProps> = ({
   const displayName = toDisplayName(field.field_name);
 
   return (
-    <li key={field.full} className="px-2 pt-2">
+    <li
+      data-testid={`cdave-row-field-${displayName}`}
```

Comment:
cdave may have to come after row-field..
does this adhere the naming convention that we're following?
https://gdc-ctds.atlassian.net/browse/QA-1888